src/components/products/filtersSection/filtersSection.tsx

```diff
@@ -0,0 +1,80 @@
+import { useSearchParams } from "react-router-dom";
+
+import SectionWrapper from "@/elements/sectionWrapper/sectionWrapper";
+import SelectFilterWrapper from "@/elements/selectFilterWrapper/selectFilterWrapper";
+import SelectControl from "@/elements/controls/select/select";
+import SelectWrapper from "@/elements/selectWrapper/selectWrapper";
+
+import RadioControl from "@/elements/controls/radio/radio";
+import RadioFilterWrapper from "@/elements/radioFilterWrapper/radioFilterWrapper";
+import styles from "./filtersSection.module.scss";
+
+interface FilterSectionProps {
+  handleCriteriaInputChange: (event: CustomEvent) => void;
```

Comment:
There is a huge amount similar callback functions. It's not ok. If filter grows you need to create more & more such callbacks.
1. The filter is specific and can be used only for GameProducts because depends on specific fields. So maybe provide single onChange callback with collected model
2. Form has $onChange event to handle any form change. So possible to handle everything in one place